borsh-rs/borsh/src/ser/mod.rs

```diff
@@ -1,17 +1,20 @@
-use std::borrow::Cow;
-use std::collections::{BTreeMap, HashMap, HashSet};
-use std::convert::TryFrom;
-use std::mem::size_of;
-use std::{io, io::Write};
+use crate::lib::*;
+
+use core::convert::TryFrom;
+use core::mem::size_of;
+use alloc::collections::BTreeMap;
+#[cfg(feature = "std")]
+use std::collections::{HashMap, HashSet};
```

Comment:
It seems that `crate::lib` was specifically created to avoid these conditional uses, right? Given we use `crate::lib` above, this import is unnecessary as far as I can tell.

Reply:
fixed